api/collection-view.yaml

```diff
@@ -334,7 +334,7 @@ functions:
       @param {Number} index
       @param {Backbone.Collection} collection
 
-   setFilter:
+    setFilter:
```

Comment:
And these spaces are really hard to get right. I'm hoping this task will tell us if they're wrong?